src/west/commands/project.py

```diff
@@ -482,17 +537,25 @@ def _projects(args, listed_must_be_cloned=True):
     # listed_must_be_cloned (default: True):
     #   If True, an error is raised if an uncloned project was listed. This
     #   only applies to projects listed explicitly on the command line.
+    #
+    # include_meta (default: False):
+    #   If True, "meta" projects (i.e. west and the manifest) may be given
+    #   in args.projects without raising errors, and are also included in the
+    #   return value if args.projects is empty.
```

Comment:
So `west list west` raises an error? That's inconsistent with `ls`, but that's just a note...

`ls .foo` works fine.

Reply:
No, `west list west` does not raise an error:

```
C:\Users\Carles\src\wncs>west list west
west           west\west          list-enhancements https://github.com/mbolivar/west.git (cloned)
```